src/utility/BinaryDownloadInfo.json

```diff
@@ -0,0 +1,87 @@
+{
+  "version": "1.0.20240109.1",
```

Comment:
sure I can add comments.

Reply:
we are replacing storage account download of b2k binaries with GitHub releases download so that this OSS tool doesn't have any azure dependencies and can be purely OSS. Plus, this extension will be decommissioned soon and thus anyone willing to use this can fork it and use it right away.